Automattic/simplenote-android
# Issue/884 app starts with white background in dark theme

### Fix
Created a splash screen theme and set it as default theme in the application.

Created a `ThemedAppCompatActivity` class to apply the actual activity theme once the activity loads. If activity theme is set in manifest, it overrides the application theme and the behaviour in the issue persists.
Modified all activities in the `Simplenote` module to extend from ThemedAppCompatActivity instead of AppCompatActivity.

### Test
1. Set app theme to 'Dark' in settings.
2. Set system theme to 'Light' (default pre Android 10/api29)
3. Close app and remove from recents.
4. Open app. 

The app window should briefly show a splash screen with white logo on blue background before opening the app in dark theme.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
